internal/config/config.go

```diff
@@ -335,6 +338,9 @@ func parsePeer(p peer) (*Peer, error) {
 	if p.ASN == 0 {
 		return nil, errors.New("missing peer ASN")
 	}
+	if p.ASN == p.MyASN && p.EBGPMultiHop {
+		return nil, errors.New("invalid ebgp-multihop set for ibgp peer")
```

Comment:
"invalid ebgp-multihop parameter set for an ibgp peer"

Reply:
done